transport/http/src/main/java/org/kaazing/gateway/transport/http/HttpStatus.java

```diff
@@ -67,7 +67,8 @@
     SERVER_BAD_GATEWAY(502, "Bad Gateway"),
     SERVER_SERVICE_UNAVAILABLE(503, "Service Unavailable"),
     SERVER_GATEWAY_TIMEOUT(504, "Gateway Timeout"),
-    SERVER_VERSION_NOT_SUPPORTED(505, "Version Not Supported");
+    SERVER_VERSION_NOT_SUPPORTED(505, "Version Not Supported"),
+    SERVER_LOOP_DETECTED(508, "Loop detected");
```

Comment:
@jitsni Hello, Jitu. Would you suggest using a different error code and not [508](https://tools.ietf.org/html/rfc5842#section-7.2)?